source/_data/SymbioticLab.bib

```diff
@@ -626,3 +629,25 @@ @InProceedings{kayak:nsdi21
     How cloud applications should interact with their data re-mains an active area of research. Over the last decade, manyhave suggested relying on a key-value (KV) interface to inter-act with data stored in remote storage servers, while othershave vouched for the benefits of using remote procedure call(RPC). Instead of choosing one over the other, in this paper,we observe that an ideal solution must adaptively combineboth of them in order to maximize throughput while meetingapplication latency requirements. To this end, we proposea new system called Kayak that proactively adjusts the rateof requests and the fraction of requests to be executed usingRPC or KV, all in a fully decentralized and self-regulated man-ner. We theoretically prove that Kayak can quickly convergeto the optimal parameters.  We implement a system proto-type of Kayak. Our evaluations show that Kayak achievessub-second convergence  and improves  overall throughputby 32.5%-63.4% for compute-intensive workloads and upto 12.2% for non-compute-intensive and transactional work-loads over the state-of-the-art.
   }
 }
+
+@InProceedings{fluid:mlsys21,
+  author    = {Peifeng Yu*, Jiachen Liu* and Mosharaf Chowdhury},
```

Comment:
Sadly there's no standard way to show equal contribution in bibtex... Adding `*` after the name will probably break standard styling/sorting/etc.

Reply:
http://www.hansenlab.org/cv_bibliography_tex
Will
  author+an =	 {1=highlight; 2=highlight}
works?

Reply:
Probably. Let's do this. It's not rendered on the website, though. I need to implement that. But if someone copies the bibtex, it should work in their pdf.